ovn_k8s/modes/overlay.py

```diff
@@ -247,6 +250,11 @@ def _update_vip(self, service_data, ips):
                     self._create_load_balancer_vip(service_type, gateway_ip,
                                                    ips, port, target_port,
                                                    protocol)
+            elif external_ips:
+                for external_ip in external_ips:
+                    self._create_load_balancer_vip("NodePort", external_ip,
+                                                   ips, port, target_port,
+                                                   protocol)
```

Comment:
I guess, you mean when a service is modified. Hmmm. I guess, we should support it. Let me look at this.